pytket_qir/gatesets/base.py

```diff
@@ -38,7 +38,6 @@ class OpName(Enum):
     AND = "and"
     OR = "or"
     XOR = "xor"
-    WASM = "wasm"
```

Comment:
Is this not needed anymore?

Reply:
No. This name is now dynamically created [here](https://github.com/CQCL/pytket-qir/blob/90b44b57a05a0b3f585b901148f2b9fed13c80e0/pytket_qir/generator.py#L286).